configure.ac

```diff
@@ -298,8 +298,8 @@ if test "$enable_subids" != "no"; then
 	dnl
 	dnl FIXME: check if 32 bit UIDs/GIDs are supported by libc
 	dnl
-	AC_CHECK_SIZEOF([uid_t],[], [#include "sys/types.h"])
-	AC_CHECK_SIZEOF([gid_t],[], [#include "sys/types.h"])
+	AC_CHECK_SIZEOF([uid_t],[], [[#include "sys/types.h"]])
+	AC_CHECK_SIZEOF([gid_t],[], [[#include "sys/types.h"]])
```

Comment:
Does this mean that the existing code was bogus?  Could you please clarify how this changes?

(Again, I'm an autotools inept here; need a lot of explanation.)